packages/ruleset/test/no-nullable-properties.test.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright 2017 - 2023 IBM Corporation.
+ * SPDX-License-Identifier: Apache2.0
+ */
+
+const { makeCopy, rootDocument, testRule, severityCodes } = require('./utils');
+const { noNullableProperties } = require('../src/rules');
+
+const rule = noNullableProperties;
+const ruleId = 'ibm-no-nullable-properties';
+const expectedSeverity = severityCodes.warning;
+const expectedMsg =
+  'nullable should be used only within JSON merge-patch request body schema properties';
+
+describe(`Spectral rule: ${ruleId}`, () => {
+  describe('Should not yield errors', () => {
+    it('Clean spec', async () => {
+      // rootDocument contains a nullable property in the CarPatch schema.
+      const results = await testRule(ruleId, rule, rootDocument);
+      expect(results).toHaveLength(0);
+    });
+    it('nullable specified without a correspondiing type value', async () => {
```

Comment:
🙂 

```suggestion
    it('nullable specified without a corresponding type value', async () => {
```

Reply:
We'll have to start calling you Norbert :)